packages/large-response-middleware/src/__tests__/test-setup.ts

```diff
@@ -1,2 +1,3 @@
 process.env.STAGE = 'test';
-process.env.AWS_ENDPOINT = process.env.AWS_ENDPOINT || 'http://localhost:4566';
\ No newline at end of file
+process.env.AWS_ENDPOINT = process.env.AWS_ENDPOINT || 'http://localhost:4566';
+process.env.AWS_SUPPRESS_MAINTENANCE_MODE_MESSAGE = 'true';
```

Comment:
what this for?

Reply:
to remove some warnings from still using aws sdk v2 here, something to change later for sure!